pkg/mod/fs_test.go

```diff
@@ -38,68 +40,84 @@ func TestOpenLocalFile(t *testing.T) {
 }
 
 func TestOpenLocalFileFailed(t *testing.T) {
-	t.Parallel()
+	SyslModules = false
+	defer func() { SyslModules = true }()
 
 	filename := "wrong.sysl"
 	_, memfs := syslutil.WriteToMemOverlayFs("/")
 	fs := NewFs(memfs, "../../tests/")
 	f, err := fs.Open(filename)
 	assert.Nil(t, f)
-	assert.Equal(t, fmt.Sprintf("%s not found", filepath.Join("../../tests/", filename)), err.Error())
+	assert.Equal(t, fmt.Sprintf("%s not found", filename), err.Error())
 }
 
 func TestOpenRemoteFile(t *testing.T) {
-	t.Parallel()
+	fs := afero.NewOsFs()
+	_, err := fs.Create("go.mod")
+	assert.NoError(t, err)
+	defer removeFile(t, fs, "go.mod")
 
 	filename := "github.com/anz-bank/sysl/demo/examples/Modules/deps.sysl"
 	_, memfs := syslutil.WriteToMemOverlayFs("/")
-	fs := NewFs(memfs, "")
-	f, err := fs.Open(filename)
+	mfs := NewFs(memfs, "")
+	f, err := mfs.Open(filename)
```

Comment:
nit: There's a lot of common structure between these tests. It might be worth seeing if there's a clean refactoring.